.changeset/great-spies-exist.md

```diff
@@ -0,0 +1,51 @@
+---
+'@xstate/graph': major
+---
+
+Coverage can now be obtained from a tested test model via `testModel.getCoverage(...)`:
+
+```js
+import { configure, createTestModel } from '@xstate/test';
+import {
+  coversAllStates,
+  coversAllTransitions
+} from '@xstate/test/lib/coverage';
+
+// ...
+
+const testModel = createTestModel(someMachine);
+
+const plans = testModel.getShortestPlans();
+
+for (const plan of plans) {
+  await testModel.testPlan(plan);
+}
+
+// Returns default coverage:
+// - state coverage
+// - transition coverage
+const coverage = testModel.getCoverage();
+
+// Returns state coverage
+const stateCoverage = testModel.getCoverage([coversAllStates()]);
+
+// Returns state coverage
+const stateCoverage = testModel.getCoverage([coversAllStates()]);
+
+// Throws error for default coverage:
+// - state coverage
+// - transition coverage
+testModel.testCoverage();
+
+// Throws error if state coverage not met
+testModel.testCoverage([stateCoverage]);
+
+// Filters state coverage
+testModel.testCoverage(
+  coversAllStates({
+    filter: (stateNode) => {
+      return stateNode.key !== 'somePassedState';
+    }
+  })
+);
+```
```

Comment:
we should also update (or at least note down that we need to update it) the README of the `@xstate/test` package

Reply:
I'll handle this as part of the docs rewrite when we finalise the API, doesn't need to be included in this PR.